### PR TITLE
Pythonization for RooAbsReal::plotOn() RooAbsPdf::fitTo() functions.

### PR DESCRIPTION
**Pull request by Harshalzzzzzzz**
@guitargeek PR for pythonization and test for RooAbsReal::plotOn() Pythonization

### CODE REVIEW

**Comment by guitargeek** [comment]
@phsft-bot build

File: bindings/pyroot/pythonizations/python/ROOT/pythonization/_roofit/_roopythonize.py [hunk 1]
```diff
@@ -1,13 +1,3 @@
-# Author: Hinnerk C. Schmidt 02/2021
-
-################################################################################
-# Copyright (C) 1995-2020, Rene Brun and Fons Rademakers.                      #
-# All rights reserved.                                                         #
-#                                                                              #
-# For the licensing terms see $ROOTSYS/LICENSE.                                #
-# For the list of contributors see $ROOTSYS/README/CREDITS.                    #
-################################################################################
-
```
Comment (Axel-Naumann) [inline bindings/pyroot/pythonizations/python/ROOT/pythonization/_roofit/_roopythonize.py:10]: Why did you move the file? Can you keep the copyright section, please?

**Review by Axel-Naumann — CHANGES_REQUESTED** [review]

File: bindings/pyroot/pythonizations/python/ROOT/pythonization/_roofit/_roopythonize.py [hunk 2]
```diff
@@ -44,15 +34,39 @@ def _fitTo(self, *args, **kwargs):
         return self._OriginalFitTo(*nargs)
 
 
+def _plotOn(self, *args, **kwargs):
+    """
+    Docstring
```
Comment (Axel-Naumann) [inline bindings/pyroot/pythonizations/python/ROOT/pythonization/_roofit/_roopythonize.py:39]: Can you improve the documentation?

File: bindings/pyroot/pythonizations/python/ROOT/pythonization/_roofit/_roopythonize.py [hunk 3]
```diff
@@ -44,15 +44,39 @@ def _fitTo(self, *args, **kwargs):
...
```
Comment (guitargeek) [inline bindings/pyroot/pythonizations/python/ROOT/pythonization/_roofit/_roopythonize.py:49]: I know you copy-pasted this ```Docstring``` line from the existing `_fitTo` function, but as Axel said this is not very meaningful just like that with no information.

For the existing `fitTo` function, it was there because we wanted to see if the docstring mechanism works in principle, i.e. that it gets shown when you type `?RooAbsReal.plotOn` in an ipython shell. Well, that worked, but we still have not figured out if we need such a docstring and what it should contain.

The problem is that the docstring should in principle document the function in all detail, but a detailed documentation already exists from the C++ code via doxygen. However, both the duplication of documentation and leaving the docstring empty are not good solutions for me.

So in conclusion, it's not clear yet what the docstring should contain yet, but I'll make sure we converge to a good solution before the next release. For this very welcome first PR of yours, I'm fine with this placeholder.

**Comment by guitargeek** [comment]
@phsft-bot build

**Review by Axel-Naumann — APPROVED** [review]
I'll leave the final review to @guitargeek !

File: bindings/pyroot/pythonizations/python/ROOT/pythonization/_roofit/_roopythonize.py (hunk 3 above)
Comment (Harshalzzzzzzz) [reply bindings/pyroot/pythonizations/python/ROOT/pythonization/_roofit/_roopythonize.py:49]: Dear Mentor,

Sorry about this, when I was asked to change the documentation, I assumed it should be consistent with the one for _fitTo and since there was no input as such  about it from your end in the PR I had submitted as pre exercise, I assumed it is supposed to be that way. Is there anything I can do to improve this ?
Thanks

**Review by guitargeek — APPROVED** [review]
Thanks! We still have a lot to figure out as we go on, but this is a good start!

**Comment by etejedor** [comment]
The comments I made in #7314 also address the changes in this PR, to be discussed with @guitargeek .
